server/opts.go

```diff
@@ -1182,6 +1182,8 @@ func (o *Options) processConfigFileLine(k string, v any, errors *[]error, warnin
 		o.MaxTracedMsgLen = int(v.(int64))
 	case "max_subscriptions", "max_subs":
 		o.MaxSubs = int(v.(int64))
+	case "max_closed_clients":
```

Comment:
We don't have really a rule on where to place it, but it looks like it is in between 2 properties related to subscriptions, so I would move it either down by one, or like I tend to do, add the new ones at the end of the `switch` (before the `default`) to help when it is time to document (easier to find the new ones ;-)).